test/functional/c/test_magnetic_dataset.cpp

```diff
@@ -330,4 +330,33 @@ TEST(MagneticDataset, test_with_slightly_distorted_positions) {
     spg_free_magnetic_dataset(dataset);
 }
 
+TEST(MagneticDataset, test_with_slightly_distorted_positions2) {
```

Comment:
```suggestion
TEST(MagneticDataset, test_failure_with_slightly_distorted_positions) {
```

Any other comments to distinguish from the test above? Particularly about `too large symprec for this input` -> cannot distinguish...?

On another note, is it possible to round the numbers to more human-readable values, maybe using `std::math` where needed? Then on the positions you can add the numerical noise so that we can see it w.r.t. `symprec`

Reply:
I've added a comment but it's hard to distinguish from the other tests...

> is it possible to round the numbers to more human-readable values, maybe using std::math where needed?

No, this structure was randomly generated.

Reply:
> No, this structure was randomly generated.

Then can you add a comment that the structure is randomly generated or provided from an issue link?